docs/root/operations/cli.rst

```diff
@@ -192,6 +192,20 @@ following are the command line options that Envoy supports.
   more. The administration interface usage is similar. Please see `Administration interface 
   <https://www.envoyproxy.io/docs/envoy/latest/operations/admin>`_ for more detail.
 
+.. option:: --socket-path <path string>
+
+  *(optional)* The output file path to the socket address for:ref:`hot restart <arch_overview_hot_restart>`.
+  Default to "@envoy_domain_socket" which will be created in the abstract namespace. Prefix _{role}_{id}
+  is appended to provide name.
```

Comment:
Please also document that all envoy processes wanting to participate in hot-restart together must use the same value for this option.

Reply:
Added in docs.